forest/src/cli/chain_cmd.rs

```diff
@@ -50,26 +51,28 @@ pub enum ChainCommands {
     },
 
     /// Prints out the genesis tipset
-    #[structopt(about = "Prints genesis tipset", help = "Prints genesis tipset")]
+    #[structopt(about = "Prints genesis tipset")]
     Genesis,
 
     /// Prints out the canonical head of the chain
-    #[structopt(about = "Print chain head", help = "Print chain head")]
+    #[structopt(about = "Print chain head")]
```

Comment:
In general, do we need those `about`s? To my understanding, they apply only to the top-level app.

Reply:
We don't need the abouts if we already have doc comments.

```
/// Prints out the genesis tipset
#[structopt(about = "Prints genesis tipset")]
Genesis,
```
So in this case it's redundant. Let's keep the more up to date comment/about and put it in a doc comment for all `#[structopt(about =`.